variables.tf

```diff
@@ -0,0 +1,81 @@
+variable "name" {
+  description = "The name of the app to be used in labels."
+  default     = "app"
+}
+
+variable "namespace" {
+  description = "The namespace to be used in labels."
+  default     = "global"
+}
+
+variable "delimiter" {
+  description = "The delimiter to be used in labels."
+  default     = "-"
+}
+
+variable "stage" {
+  description = "Stage to be used in labels."
+  default     = "default"
+}
+
+variable "attributes" {
+  type    = "list"
+  default = []
+}
+
+variable "tags" {
+  type    = "map"
+  default = {}
+}
+
+variable "vpc_id" {
+  description = "The VPC id where resources are created."
+}
+
+variable "alb_arn" {
+  description = "The ALB arn where listener and target group will be created."
+}
+
+variable "private_subnet_ids" {
+  description = "Private subnet ids."
+  type        = "list"
+}
+
+variable "security_group_ids" {
+  description = "Security group IDs to allow in Service network_configuration."
+  type        = "list"
+}
+
+#variable "acm_arn" {
```

Comment:
I think this should just get dropped since the ALB stuff will get provisioned outside of this module.

Reply:
True - cert will be terminated at ALB.